pkg/log/wrapper.go

```diff
@@ -46,37 +47,32 @@ type Logger struct {
 	level Level // Level specifies a level of verbosity for V logs.
 }
 
-// LogDepth logs a message with the given level and depth.
-func (l *Logger) LogDepth(calldepth int, level Level, msg string, args ...any) {
```

Comment:
What does this `calldepth` now use instead?

`source` is not the correct log location.

``` console
{"time":"2023-05-04T14:01:53.85924416Z","level":"DEBUG","source":"/home/runner/work/kwok/kwok/pkg/log/wrapper.go:57","msg":"Load config","path":[],"err":"empty config"}
{"time":"2023-05-04T14:01:53.859318761Z","level":"DEBUG","source":"/home/runner/work/kwok/kwok/pkg/log/wrapper.go:57","msg":"No configuration","kind":"KwokctlConfiguration"}
{"time":"2023-05-04T14:01:53.859670064Z","level":"INFO","source":"/home/runner/work/kwok/kwok/pkg/log/wrapper.go:62","msg":"Cluster is creating","cluster":"cluster-binary-1-27-1"}
```

Reply:
I believe I have this fixed now, its unfortunate but had to replicate some of the slog.Logger logic into our own wrapper.